src/python/tests/core/bot/tasks/corpus_pruning_task_data/fuchsia/AA

```diff
@@ -0,0 +1 @@
+AA
```

Comment:
nit: add some more duplicates to ensure they get removed? Or are AA and CC expected to reach the same coverage? If they reach the same coverage, let's just make the file contents the same for determinism.

Reply:
huh, while fixing this i noticed something curious!

i would expect AA and CC to reach the same coverage, yes.  but i would *also* expect BB to reach the same coverage.  and yet: when i have (AA1, AA2, H, HI!), i only get *one* corpus unit ("H", as expected), but when i add BB to the mix (e.g. AA1, AA2, B, H, HI!), i get *two* corpus units ("H", "BB").

this is unintuitive and weird and making me wonder if i'm misunderstanding something.  do you have any hunches?  the fuzzer we're using is here: https://fuchsia.googlesource.com/fuchsia/+/refs/heads/master/examples/fuzzer/trap.cc

Reply:
As discussed, there may be some non determinism here with the merge algorithm or possibly the target. Let's just go with a simpler input which gives us the expected result.

Reply:
discussed offline--fixed by just having AA1 and AA2